frontend/src/metabase/lib/time.js

```diff
@@ -71,3 +72,48 @@ export function parseTime(value) {
     return moment.utc(value);
   }
 }
+
+export function formatHourAMPM(hour) {
+  if (hour > 12) {
+    const newHour = hour - 12;
+    return t`${newHour}:00 PM`;
+  } else if (hour === 0) {
+    return t`12:00 AM`;
```

Comment:
Doesn't it need 24h-noon as 12pm too?
```suggestion
  } else if (hour === 0) {
    return t`12:00 AM`;
  } else if (hour === 12) {
    return t`12:00 PM`;
```